UPGRADE.md

```diff
@@ -18,22 +18,50 @@ RAILS_ENV=production rake sharetribe:generate_customization_stylesheets_immediat
+
+To enable the warning in Heroku:
+
+```bash
+heroku config:set next_maintenance_at="2016-04-29 17:15:00 +0000" --app=<your app name>
+```
+
+See instructions how to set application in [maintenance mode in Heroku](https://devcenter.heroku.com/articles/maintenance-mode).
+
 ## Unreleased
 
+### Separate CSS compilation workers
+
 This release adds a new Delayed Job queue "css_compile". All CSS compilations during the deployment are added to this queue. However, CSS compilations triggered from the admin UI do NOT go into this queue, instead they are added to the "default" queue.
 
 A new worker is added to the Procfile to work for the new queue. If you're hosting in Heroku, you will see a new worker there.
 
-This change doesn't require any changes if you are compiling the stylesheets synchronously using the `rake sharetribe:generate_customization_stylesheets_immediately` command during the deployment. However, if you are compiling the stylesheets asynchronously using the `rake sharetribe:generate_customization_stylesheets` command, then you need to make sure that you have at least one worker working for the "css_compile" queue.
+This change doesn't require any changes, if you are compiling the stylesheets synchronously using the `rake sharetribe:generate_customization_stylesheets_immediately` command during the deployment. However, if you are compiling the stylesheets asynchronously using the `rake sharetribe:generate_customization_stylesheets` command, then you need to make sure that you have at least one worker working for the "css_compile" queue.
+
+### React on Rails build environment
 
+React on Rails build environment is added in this release. This means that build environment needs to have `node` set up. With Heroku this can be set with `heroku buildpacks:add --index 1 heroku/nodejs`. For other environments - see [npm instructions](https://docs.npmjs.com/getting-started/installing-node), [nvm](https://github.com/creationix/nvm), or [n](https://github.com/tj/n). In addition, production environments should have `NODE_ENV=production` set.
 
-React on Rails build environment is added with this change. This means that build environment needs to have node set up. With Heroku this can be set with ```heroku buildpacks:add --index 1 heroku/nodejs```. For other environments - see [npm instructions](https://docs.npmjs.com/getting-started/installing-node), [nvm](https://github.com/creationix/nvm), or [n](https://github.com/tj/n). In addition, production environments should have ```NODE_ENV=production`` set.
+After bundle install, you should also install `npm` packages:
 
-After bundle install, you should also install npm packages:
 ```bash
 npm install
 ```
 
+### User account migrations
+
+You can ignore this section if you are running only one marketplace in your Sharetribe instance.
```

Comment:
Does the current OS version support multiple marketplaces in one instance? And if it does, has it been documented? Should we leave this chapter out if having multiple marketplaces is not an _official_ feature?